e2e/helpers/rpc.helper.ts

```diff
@@ -0,0 +1,139 @@
+import { readFileSync } from 'fs';
+import * as yaml from 'js-yaml';
+import axios, { AxiosRequestConfig } from 'axios';
+
+type PartialUtxo = {
+    value: number;
+    scriptPubKey: {
+        address: string;
+    };
+};
+
+export class BitcoinRPCUtil {
+    private readonly url: string;
+    private readonly axiosConfig: AxiosRequestConfig;
+
+    constructor(configPath = './config/e2e.config.yaml') {
+        const config = yaml.load(readFileSync(configPath, 'utf8')) as Record<
+            string,
+            any
+        >;
+        const user = config.bitcoinCore.rpcUser;
+        const password = config.bitcoinCore.rpcPass;
+        const host = config.bitcoinCore.rpcHost;
+        const port = config.bitcoinCore.rpcPort;
+        const protocol = config.bitcoinCore.protocol;
+        this.url = `${protocol}://${host}:${port}/`;
+        this.axiosConfig = {
+            method: 'POST',
+            auth: {
+                username: user,
+                password: password,
+            },
+        };
+    }
+
+    public async request(config: AxiosRequestConfig): Promise<any> {
+        try {
+            const response = await axios.request({
+                ...this.axiosConfig,
+                ...config,
+                url: this.url,
+            });
+            return response.data?.result;
+        } catch (error) {
+            throw new Error(
+                `Request failed with status code ${error.response.status}`,
+            );
+        }
+    }
+
+    getBlockHeight(): Promise<number> {
+        return this.request({
+            data: {
+                jsonrpc: '1.0',
+                id: 'silent_payment_indexer',
```

Comment:
Is `jsonrpc` and `id` a required param? If yes, then add it to all calls, otherwise remove it from everywhere.